lib/word_count_cases.rb

```diff
@@ -4,7 +4,7 @@ def name
   end
 
   def object_under_test
-    %Q(Phrase.new("#{input}"))
+    %Q(Phrase.new(#{input.to_json}))
```

Comment:
Sorry to be nitpicky, but..
I think I prefer using `inspect` here, since it's not REALLY json that we want.